lib/Saml2/Utils.php

```diff
@@ -968,10 +968,9 @@ public static function deleteLocalSession()
     {
 
         if (OneLogin_Saml2_Utils::isSessionStarted()) {
+            session_unset();
```

Comment:
What about:

if (OneLogin_Saml2_Utils::isSessionStarted()) {
    session_unset();
    session_destroy();
} else {
    session_unset();
}

Reply:
No problem, however I suggest to put the unset before the destroy without the else:

```php
session_unset();

if (OneLogin_Saml2_Utils::isSessionStarted()) {
    session_destroy();
}
```

Reply:
Yeah, modify the PR and I will merge, thanks!

Reply:
Yes done! 💪